mimesis/providers/industry.py

```diff
@@ -0,0 +1,55 @@
+from mimesis.providers.base import BaseDataProvider
+
+__all__ = ['Industry']
+
+
+class Industry(BaseDataProvider):
+
+    def __init__(self, *args, **kwargs):
+        """Initialize attributes.
+
+        :param args: Arguments.
+        :param kwargs: Keyword arguments.
+        """
+        super().__init__(*args, **kwargs)
+
+    def api_well_number(self) -> str:
+        """"Consists of:
```

Comment:
We use a different style of documentation. Please,  check out methods in other providers.